src/delay.rs

```diff
@@ -92,21 +92,22 @@ impl From<TickType> for Option<Duration> {
 /// trigger.
 pub struct Ets;
 
-// No longer available in the generated bindings for ESP-IDF 5
+// Not available in the generated bindings for ESP-IDF 4
```

Comment:
You have changed the comment to read exactly the opposite of what the code actually does. Why?

Reply:
That's how I read the code.
"not available in IDF 4" -> '#[cfg(not(...=4'.

I just guessed that this was a comment typo. If it's not, then I don't understand the comment.

Reply:
No - the `cfg` says "**enable** the code below for ESP IDF != 4". Which is the same as "enable the code below for ESP IDF > 4" (as we don't support ESP IDF 3, and `cfg`s only work for equality.

In ESP IDF 4 the manual definition of this function signature is not necessary, as it is available in the `bindings.rs` file that `esp-idf-sys` generates. It is no longer available in `bindings.rs` since ESP IDF 5. Might be that they moved it to another header that we don't include (yet) but I couldn't find it.

Reply:
Ok, I get it now.

>In ESP IDF 4 the manual definition of this function signature is not necessary, as it is available in the bindings.rs

That is the part I did not get from the comment.
I understood the comment as such that the whole feature you are using is not available. Didn't make sense to me to define a binding for something that is not available.

What about this:
```
// This binding is no longer available in the generated bindings for ESP-IDF 5.
// The function itself is still available in ESP-IDF 5. Therefore, we define the binding here.
```

Reply:
Your suggestion merged, thanks!